components/camel-salesforce/camel-salesforce-component/src/main/java/org/apache/camel/component/salesforce/api/dto/SObject.java

```diff
@@ -43,6 +43,7 @@
     private Boolean undeletable;
     private Boolean triggerable;
     private Boolean compactLayoutable;
+    private Boolean mruEnabled;
```

Comment:
This one came back as a deserialization error when setting the Salesforce API version to 37.0.